regression/goto-analyzer/intervals11/intervals11.c

```diff
@@ -0,0 +1,43 @@
+#include <assert.h>
+const int xLen = 10;
+const int Alen = 2;
+const int Blen = 1;
+float nondet_float();
+int main() {
```

Comment:
It would be nice if regression tests also somewhat conformed with coding standards...